tests/coalaCITest.py

```diff
@@ -105,10 +105,8 @@ def test_coala_delete_orig(self):
              make_temp(suffix='.coafile', prefix='', dir=tempdir) as coafile,\
              make_temp(dir=tempdir) as unrelated_file:
             orig_file.close()
-            with open(coafile, mode="w") as file:
-                file.write("files = something\n")
 
-            execute_coala(coala_ci.main, "coala-ci",
-                          "-c", re.escape(coafile))
+            execute_coala(coala_ci.main, "coala-ci", "-S",
+                          "project_dir=" + os.path.dirname(coafile))
```

Comment:
I wasn't sure. The deleted line has `re.escape(coafile)`. Apparently it does handle escaping (as the tests pass)